### DISCUSSION TITLE
Implement net.mail with support for the SMTP, POP3 and IMAP protocols

### DISCUSSION THREAD

**Discussion by changrui** [vlang/v#11634] [Ideas]
I think V needs this module, but i do not do it. This module will implement `smtp`, `pop3` and `imap`.

**Comment by ITwrx** [comment]
net has smtp now. If imap were implemented, i would have an excuse to learn some v and try to write a simple mail client. :) nim doesn't have imap either, or i probably would have tried with that. just think, vlang could be cooler than nim. :)

**Reply by JalonSolov** [reply]
It always was, but yes, it would be nice to have imap, too. :smile:

**Reply by ITwrx** [reply]
if i get time i'll see if i can contribute anything. [edited for brevity/keep on topic]
